visualization/include/pcl/visualization/vtk/pcl_vtk_compatibility.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2020-, Open Perception
+ *
+ *  All rights reserved
+ */
+
+#include <vtkCellArray.h>
+
+#ifdef VTK_CELL_ARRAY_V2
+  using vtkCellPtsPtr = vtkIdType const*;
```

Comment:
You're using this same macro in 2 files: 1 each in io and surface module (search for `vtkIdType const`)

Reply:
Yeah, its only used once in each module and there are no common module, which depends on VTK to move such a definition to.
Surface is independent on io and vice versa. So I think its easiest to let those two definitions stay?

Reply:
Sure. Makes sense.